src/ua/gorshkov/hw8/Main.java

```diff
@@ -0,0 +1,21 @@
+package ua.gorshkov.hw8;
+
+public class Main {
+    public static void main(String[] args) {
+        Animal dog = new Animal("meat", 10);
+        Animal cat = new Animal("mouse", 12);
+        Animal bee = new Animal("honey", 1);
+        int biggestTimeOfSleeping = getBiggestSleepingTime(dog, cat, bee);
+        System.out.println("Biggest sleeping time = " + biggestTimeOfSleeping);
```

Comment:
You had to find an animal with the biggest time of sleeping and display it's kind of food